src/jailer/src/env.rs

```diff
@@ -546,4 +558,57 @@ mod tests {
         // And changing the umask in the Python integration tests is unsafe because of pytest's
         // process management; it can't be isolated from side effects.
     }
+
+    fn skip_not_found(err: std::io::Error) -> std::io::Result<()> {
+        match err.kind() {
+            std::io::ErrorKind::NotFound => Ok(()),
+            _ => Err(err),
+        }
+    }
+
+    fn get_major(dev: u64) -> u32 {
+        unsafe { libc::major(dev) }
+    }
+
+    fn get_minor(dev: u64) -> u32 {
+        unsafe { libc::minor(dev) }
+    }
+
+    #[test]
+    fn test_mknod_and_own_dev() {
+        use std::os::unix::fs::{FileTypeExt, PermissionsExt};
```

Comment:
hmm, `PermissionsExt` should've been removed, not `MetadataExt`.

Reply:
I've done both, since `std::os::linux::fs::MetadataExt` is imported in the tests module.

The only additional change was [`rdev()`](https://doc.rust-lang.org/std/os/unix/fs/trait.MetadataExt.html#tymethod.rdev) → [`st_rdev()`](https://doc.rust-lang.org/std/os/linux/fs/trait.MetadataExt.html#tymethod.st_rdev) conversion.

Reply:
Yep, I didn't suggest to remove that too because I kind of prefer to use these from `unix`, not `linux`.